### DISCUSSION TITLE
Could you please let us know when we can expect a release version that supports psr/log 1.* and psr/log 2.*?

### DISCUSSION THREAD

**Discussion by michelvambattu** [netsuitephp/netsuite-php#174] [Q&A]
Could you please let us know when we can expect a release version that supports `psr/log 1.*` and` psr/log 2.*`?

Since the version `v2023.1.0 `only supports `"psr/log": "^1.0"`, it's a blocker because the current installed version of `psr/log `is `2.0` which is used by many other packages.

I see that dev-master resolves this issue by supporting multiple versions of `psr/log`. Could you please let us know when we could expect this release?
![image](https://github.com/user-attachments/assets/f4a3e2d0-db18-4fbd-a01b-a4632aff8c9e)
![image](https://github.com/user-attachments/assets/8be245c6-2bec-4889-9d0d-8287fccb9342)

**Comment by neclimdul** [comment]
@jrebs unless you've got any objects I'll take a look at pushing a tag tomorrow.

**Reply by jrebs** [reply]
Yeah that would be great. I'm away on a summer break atm, but if the change is in and just needs a release, feel free. Thanks.
